OnDemandPassengerTests/OnDemandPassengerTests-Bridging-Header.h

```diff
@@ -0,0 +1,9 @@
+//
```

Comment:
Why did you add this file? Bridging header is used for reaching Objective-C classes from Swift classes. Our project is already based on Objective-C.